diff -ur IO-AIO-4.4.orig/AIO.pm IO-AIO-4.4/AIO.pm
--- IO-AIO-4.4.orig/AIO.pm	2018-02-20 05:44:15.000000000 +0000
+++ IO-AIO-4.4/AIO.pm	2018-03-08 12:43:23.252765304 +0000
@@ -173,7 +173,7 @@
 use base 'Exporter';
 
 BEGIN {
-   our $VERSION = 4.4;
+   our $VERSION = 4.40;
 
    our @AIO_REQ = qw(aio_sendfile aio_seek aio_read aio_write aio_open aio_close
                      aio_stat aio_lstat aio_unlink aio_rmdir aio_readdir aio_readdirx
diff -ur IO-AIO-4.4.orig/Changes IO-AIO-4.4/Changes
--- IO-AIO-4.4.orig/Changes	2018-02-20 06:54:29.000000000 +0000
+++ IO-AIO-4.4/Changes	2018-03-08 12:43:12.604765424 +0000
@@ -34,7 +34,7 @@
 TODO: define useful rules for [[$wd, $path], $path] &c and implement them
 TODO: https://github.com/coreutils/coreutils/blob/master/src/stat.c maybe IO::AIO::FSID
 
-4.4   Tue Feb 20 07:54:12 CET 2018
+4.40  Tue Feb 20 07:54:12 CET 2018
 	- new aio_slurp request.
         - add (experimental) IO::AIO::eventfd and timerfd* function.
 	- add (experimental) IO::AIO::get_fdlimit and IO::AIO::min_fdlimit.
diff -ur IO-AIO-4.4.orig/META.json IO-AIO-4.4/META.json
--- IO-AIO-4.4.orig/META.json	2018-02-20 06:54:40.000000000 +0000
+++ IO-AIO-4.4/META.json	2018-03-08 12:43:18.204765361 +0000
@@ -38,6 +38,6 @@
       }
    },
    "release_status" : "stable",
-   "version" : 4.4,
+   "version" : 4.40,
    "x_serialization_backend" : "JSON::PP version 2.27300"
 }
diff -ur IO-AIO-4.4.orig/META.yml IO-AIO-4.4/META.yml
--- IO-AIO-4.4.orig/META.yml	2018-02-20 06:54:40.000000000 +0000
+++ IO-AIO-4.4/META.yml	2018-03-08 12:43:03.260765530 +0000
@@ -20,5 +20,5 @@
     - inc
 requires:
   common::sense: '0'
-version: 4.4
+version: 4.40
 x_serialization_backend: 'CPAN::Meta::YAML version 0.012'
